src/sentry/static/sentry/app/views/releasesV2/list/index.tsx

```diff
@@ -68,15 +65,37 @@ class ReleasesList extends AsyncView<Props, State> {
         'healthStat',
       ]),
       summaryStatsPeriod: statsPeriod,
-      per_page: 50,
-      health: 1,
+      per_page: 25,
+      health: 0,
       flatten: !sort || sort === 'date' ? 0 : 1,
     };
 
-    return [['releases', `/organizations/${organization.slug}/releases/`, {query}]];
+    return [
+      ['releasesWithoutHealth', `/organizations/${organization.slug}/releases/`, {query}],
+      [
+        'releasesWithHealth',
+        `/organizations/${organization.slug}/releases/`,
+        {query: {...query, health: 1}},
```

Comment:
Have we considered having another endpoint that fetches only the health data without having to redo the work around getting a list of releases? Perhaps a bulk GET endpoint that takes a list of versions and returns the health data?

Reply:
Yeah, and this might be something we want to try later. Generally it's a lot more work because it's conditional not just on the version but also each individual project and environment selection and we would need to pass *a lot of data* as GET parameters which might be way beyond the max payload size for URLs.